core/templates/dev/head/filters.js

```diff
@@ -342,6 +342,26 @@ oppia.filter('summarizeNonnegativeNumber', [function() {
   };
 }]);
 
+// Note that this filter removes additional new lines or <p><br></p> tags
+// at the end of the string.
+oppia.filter('removeExtraLines', [function() {
+  return function(string) {
+    if (!angular.isString(string)) {
+      return string;
+    }
+    while (1) {
+      var lastIndex = string.lastIndexOf('>');
```

Comment:
If we are only removing empty lines at the end of the string, shouldn't this index always be just the last index of the string? Or is there some strange edge case if we just use the last index of the string?